extensions/scarb-doc/src/types.rs

```diff
@@ -167,7 +170,7 @@ pub struct Constant {
 }
 
 impl Constant {
-    pub fn new(db: &dyn DefsGroup, id: ConstantId, node: &ast::ItemConstant) -> Self {
+    pub fn new(db: &dyn DocGroup, id: ConstantId, node: &ast::ItemConstant) -> Self {
```

Comment:
TBH It'd be easier if you just use `&ScarbDocDatabase` directly. At least that's the conclusion I did in LS.

Reply:
I have to redo the #1315 PR completely anyways (2137 conflicts) so I will do it there